vendor-extra/ContentPageBundle/src/DependencyInjection/Compiler/ContentHandlerPass.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentPageBundle\DependencyInjection\Compiler;
+
+use LogicException;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+use function array_keys;
+
+final class ContentHandlerPass implements CompilerPassInterface
```

Comment:
these `Passes` could do with a docblock describing how they map to the YAML schema, but maybe this is already described elsewhere